src/hooks/protections/receive_net_message.cpp

```diff
@@ -85,6 +84,7 @@ namespace big
 
 		rage::eNetMessage msgType;
 		player_ptr player;
+		rate_limiter unk_player_radio_requests{5s, 2};
```

Comment:
Why is this outside the scope of MsgRadioStationSyncRequest? Make it a static, too, otherwise the compiler will discard it.